pulp_container/app/registry_api.py

```diff
@@ -689,34 +661,26 @@ def put(self, request, path, pk=None):
         _, repository = self.get_dr_push(request, path)
 
         digest = request.query_params["digest"]
-        # Try to see if the client came back after we told it to backoff with the ``Throttled``
-        # exception. In that case we answer based on the task state, or make it backoff again.
-        # This mechanism seems to work with podman but not with docker. However we let the task run
-        # anyway, since all clients will look with a HEAD request before attemting to upload a blob
-        # again.
+        # check if the client came back after the artifact was already uploaded; in that case, we
```

Comment:
you are raising here upload 404, client does not come back, it comes back only if you issue 429.